crates/protocol/src/predeploys.rs

```diff
@@ -0,0 +1,7 @@
+//! Addresses of OP pre-deploys.
+
+use alloy_primitives::{address, Address};
+
+/// The L2 contract `L2ToL1MessagePasser`, stores commitments to withdrawal transactions.
+pub const ADDRESS_L2_TO_L1_MESSAGE_PASSER: Address =
+    address!("4200000000000000000000000000000000000016");
```

Comment:
makes it easier to search for the address
```suggestion
    address!("0x4200000000000000000000000000000000000016");
```